spec/support/database_cleaner.rb

```diff
@@ -4,11 +4,13 @@
     DatabaseCleaner.clean_with(:truncation)
```

Comment:
@maximlarionov I don't know is it really necessary, because at least latest version doesn't fix the bug